src/pages/developing/frameworks/web-components.mdx

```diff
@@ -0,0 +1,176 @@
+instead:
+
+```bash
+yarn add carbon-web-components carbon-components lit-html lit-element
+```
+
+### Basic usage
+
+Our example at [CodeSandbox](https://codesandbox.io) shows the most basic usage:
+
+[![Edit carbon-web-components](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/github/carbon-design-system/carbon-web-components/tree/master/examples/codesandbox/basic)
+
+The first thing you need is **setting up a module bundler** to resolve
+ECMAScript `import`s. Above example uses [Webpack](https://webpack.js.org). You
+can use other bundlers like [Rollup](https://rollupjs.org/) too.
+
+Once you set up a module bundler, you can start importing our component modules,
+for example:
+
+```javascript
+import 'carbon-web-components/es/components/dropdown/dropdown.js';
+import 'carbon-web-components/es/components/dropdown/dropdown-item.js';
+```
+
+Once you do that, you can use our components in the same manner as native HTML
+tags, for example:
+
+```html
+<bx-dropdown trigger-content="Select an item">
+  <bx-dropdown-item value="all">Option 1</bx-dropdown-item>
+  <bx-dropdown-item value="cloudFoundry">Option 2</bx-dropdown-item>
+  <bx-dropdown-item value="staging">Option 3</bx-dropdown-item>
+  <bx-dropdown-item value="dea">Option 4</bx-dropdown-item>
+  <bx-dropdown-item value="router">Option 5</bx-dropdown-item>
+</bx-dropdown>
+```
+
+If you just want to try our components for demonstrations and so on, you can use
+CDNs that support module mapping (for example, [JSPM](https://jspm.org)). With
+it, you can import just our modules in `<script type="module">`:
+
+```html
+<!DOCTYPE html>
+<html>
+  <head>
+    <script type="module">
+      import 'https://jspm.dev/carbon-web-components/es/components/dropdown/dropdown.js';
+      import 'https://jspm.dev/carbon-web-components/es/components/dropdown/dropdown-item.js';
+    </script>
+    <style type="text/css">
+      #app {
+        font-family: 'IBM Plex Sans', 'Helvetica Neue', Arial, sans-serif;
+        width: 300px;
+        margin: 2rem;
+      }
+
+      bx-dropdown:not(:defined),
+      bx-dropdown-item:not(:defined) {
+        visibility: hidden;
+      }
+    </style>
+  </head>
+  <body>
+    <div id="app">
+      <bx-dropdown trigger-content="Select an item">
+        <bx-dropdown-item value="all">Option 1</bx-dropdown-item>
+        <bx-dropdown-item value="cloudFoundry">Option 2</bx-dropdown-item>
+        <bx-dropdown-item value="staging">Option 3</bx-dropdown-item>
+        <bx-dropdown-item value="dea">Option 4</bx-dropdown-item>
+        <bx-dropdown-item value="router">Option 5</bx-dropdown-item>
+      </bx-dropdown>
+    </div>
+  </body>
+</html>
+```
+
+### List of available components
+
+View available web components
+[here](https://web-components.carbondesignsystem.com). Usage information is
+available in the notes provided with each story.
+
+### JavaScript framework integration
```

Comment:
Can we be a little more direct with how the web component wrappers are different than Carbon Angular, Carbon React, and Carbon Vue component libraries? It's more than just who maintains them - it's differences in APIs (e.g. we wouldn't want somebody trying to reference the Carbon React docs when actually using the web component wrapper.) It's differences in accessibility audits and tests and to some degree, differences in behavior as well.

@joshblack have an suggestion on a disclaimer that we can put in a notification box here or something so there is no confusion between framework-native implementations, and framework wrappers of the web components?

Reply:
@mattrosno I added a note in the section that highlights the difference, let me know if the wording works!

fyi @janchild

Reply:
Thanks!